localstack/services/awslambda/lambda_executors.py

```diff
@@ -699,10 +699,14 @@ class Util:
     @classmethod
     def get_java_opts(cls):
         opts = config.LAMBDA_JAVA_OPTS or ''
-        if '_debug_port_' in opts:
+        if 'address=_debug_port_' in opts:
```

Comment:
I'd keep this as it was (`if '_debug_port_' in opts:`), to be more flexible for other formats or potential future changes on how the port can be configured.